app/[locale]/training-and-placement/page.tsx

```diff
@@ -1,9 +1,1200 @@
+      alt: '',
+    },
+    {
+      href: 'https://bazaar.shopclues.com/',
+      src: 'https://nitkkr.ac.in/wp-content/uploads/2022/05/172.jpg',
+      alt: '',
+    },
+    {
+      href: 'https://www.skf.com/',
+      src: 'https://nitkkr.ac.in/wp-content/uploads/2022/05/173.jpg',
+      alt: '',
+    },
+    {
+      href: 'https://www.unominda.com/',
+      src: 'https://nitkkr.ac.in/wp-content/uploads/2022/05/174.jpg',
+      alt: '',
+    },
+    {
+      href: 'https://www.mu-sigma.com/',
+      src: 'https://nitkkr.ac.in/wp-content/uploads/2022/06/microsoft.jpg',
+      alt: '',
+    },
+    {
+      href: 'https://www.microsoft.com/',
+      src: 'https://nitkkr.ac.in/wp-content/uploads/2022/06/microsoft1.jpg',
+      alt: '',
+    },
+  ];
+
+  const iconClassName =
+    'w-12 h-12 bg-primary-700 p-2 rounded-md text-neutral-50 group-hover:bg-neutral-50 group-hover:text-primary-700';
+  const recruitersLinks = [
+    {
+      href: 'https://nitkkr.ac.in/wp-content/uploads/2021/09/Build-a-Relationship-converted_1.pdf',
+      icon: <FaGears className={iconClassName} />,
+      text: text.forrecruiters.build,
+    },
+    {
+      href: 'https://nitkkr.ac.in/wp-content/uploads/2023/11/Website-Update-_-Invitation.030823.pdf',
+      icon: <FaRegEnvelope className={iconClassName} />,
+      text: text.forrecruiters.invitaion,
+    },
+    {
+      href: 'https://nitkkr.ac.in/wp-content/uploads/2021/09/Reach-Us.pdf',
+      icon: <FaGlobeAsia className={iconClassName} />,
+      text: text.forrecruiters.reach,
+    },
+  ];
+
+  const guidelinesLinks = [
+    {
+      href: 'https://nitkkr.ac.in/wp-content/uploads/2021/09/PLACEMENT_PROTOCOL_1.pdf',
+      icon: <RiBriefcase4Line className={iconClassName} />,
+      text: text.guidelines.protocol,
+    },
+    {
+      href: 'https://nitkkr.ac.in/wp-content/uploads/2021/09/TP_Cell_Guidelines.pdf',
+      icon: <MdArticle className={iconClassName} />,
+      text: text.guidelines.tnpguidelines,
+      className: { iconClassName },
+    },
+    {
+      href: 'https://nitkkr.ac.in/wp-content/uploads/2021/09/UG-Internship-Guidelines_final-08042021.pdf',
+      icon: <MdArticle className={iconClassName} />,
+      text: text.guidelines.internguidlines,
+    },
+  ];
+
+  return (
+    <>
+      <ImageHeader
+        title={text.title}
+        headings={[
+          { label: text.headings.about, href: '#about' },
+          { label: text.headings.stats, href: '#stats' },
+          { label: text.headings.ourrecruiters, href: '#our' },
+          { label: text.headings.forrecruiters, href: '#for' },
+          { label: text.headings.guidelines, href: '#guide' },
+          {
+            label: text.headings.faq,
+            href: 'https://nitkkr.ac.in/wp-content/uploads/2021/09/Frequently_Asked_Questions.pdf',
+          },
+        ]}
+        src={
+          'https://nitkkr.ac.in/wp-content/uploads/2022/03/9-1-e1700472906139.jpg'
+        }
```

Comment:
When it's just a string, don't use the `{''}` syntax, just use `""` directly.
```suggestion
        src="https://nitkkr.ac.in/wp-content/uploads/2022/03/9-1-e1700472906139.jpg"
```

Do the same everywhere else.